### ISSUE TITLE
Add aria-label / sr-only attributes to close buttons to support accessibility

### ISSUE THREAD

**Issue by satishreddy84** [aspnetcorehero/ToastNotification#11]
The Notfy or the toasts does not have any aria-labels for the close buttons. Could you please add aria-labels or sr-only attributes to them to be accessible or screen reader friendly.

Thank you.
